syz-fuzzer/testing.go

```diff
@@ -256,6 +256,7 @@ func checkSimpleProgram(args *checkArgs, features *host.Features) error {
 	if info.Calls[0].Errno != 0 {
 		return fmt.Errorf("simple call failed: %+v\n%s", info.Calls[0], output)
 	}
+	// TODO(HerrSpace): error out if cover was requested, but shmem not enabled.
```

Comment:
Shmem is fixed per OS, right? I think such check belongs to pkg/host, if an OS claims coverage is supported, but sys/targets doesn't enable shmem, it should error.

Reply:
Also fine. I just was stuck here during debugging for a while and left a note :) I'll look into it.

Reply:
I also implemented this check now.